nifi-toolkit/nifi-toolkit-encrypt-config/src/main/groovy/org/apache/nifi/properties/ConfigEncryptionTool.groovy

```diff
@@ -722,7 +707,7 @@ class ConfigEncryptionTool {
         final PropertyEncryptor inputEncryptor = new PropertyEncryptorBuilder(existingFlowPassword).setAlgorithm(existingAlgorithm).build()
         final PropertyEncryptor outputEncryptor = new PropertyEncryptorBuilder(newFlowPassword).setAlgorithm(newAlgorithm).build()
 
-        final FlowEncryptor flowEncryptor = new StandardFlowEncryptor()
+        final FlowEncryptor flowEncryptor = new XmlFlowEncryptor()
```

Comment:
I may not have communicated clearly in the previous comment, but `XmlFlowEncryptor` can be removed and `StandardFlowEncryptor` should be used in this location.

Reply:
In this case I am not 100% sure if I get your intent. `StandardFlowEncryptor` is (actually: was) a just a composite implementation which helped NiFi decide is the `XmlFlowEncryptor` or the `JsonFlowEncryptor` needs to be used. As `XmlFlowEncryptor` is not there anymore `StandardFlowEncryptor` was deleted as well. In the other hand this toolkit method needs an encryptor capable to work with XML, let it be `XmlFlowEncryptor` directly or `StandardFlowEncryptor` (which will call `XmlFlowEncryptor` in the background). So if we are willing to keep providing this particular service (`migrateFlowXmlContent`) we need the `XmlFlowEncryptor` in place. If this is your aim, I would limit it's scope to the toolkit (I added back this way) and would not add back the composite (`StandardFlowEncryptor`) as it provides no feature during the standard run of NiFi

Reply:
Thanks for the reply and clarifying the implementations. In this case, it looks like the implementation should be changed to the JsonFlowEncryptor. The `migrateFlowXmlContent()` method can be renamed to `migrateFlowContent()` and that seems sufficient for this PR. How does that sound?